code/datums/flock/flock.dm

```diff
@@ -68,7 +68,8 @@ var/flock_signal_unleashed = FALSE
 		src.unlockableStructures += new DT(src)
 
 /datum/flock/ui_status(mob/user)
-	return istype(user, /mob/living/intangible/flock/flockmind) || tgui_admin_state.can_use_topic(src, user)
+	if(istype(user, /mob/living/intangible/flock/flockmind) || tgui_admin_state.can_use_topic(src, user))
+		return UI_INTERACTIVE
```

Comment:
Currently works for Flockdrones but will this prevent that?

Reply:
well it's the same condition, it just returns 2 rather than 1 now

Reply:
Okay, not really familiar was just asking to be safe

Reply:
okay I spooked myself and checked and it does work 👍